erehwon/profiles/admin.py

```diff
@@ -1,10 +1,18 @@
 from django.contrib import admin
+from custom_user.admin import EmailUserAdmin
 
 # Register your models here.
-from .models import Project, Idea, Message, CallForAction, ErehwonUser
+from .models import Project, Idea, Message, CallForAction
+
+# class ErehwonUserAdmin(EmailUserAdmin):
+#     """
+#     You can customize the interface of your model here.
+#     """
+#     pass
+
 
 admin.site.register(Project)
 admin.site.register(Idea)
 admin.site.register(Message)
 admin.site.register(CallForAction)
-admin.site.register(ErehwonUser)
+# admin.site.register(ErehwonUser, ErehwonUserAdmin)
```

Comment:
this is required to be present, you wont be able to see any users in the admin otherwise.